python-sdk/src/astro/lineage/extractor.py

```diff
@@ -37,6 +37,7 @@ def get_operator_classnames(cls) -> list[str]:
             "LoadFileOperator",
             "MergeOperator",
             "TransformOperator",
+            "DataframeOperator",
```

Comment:
Look like this list is in sorted order

Reply:
addressed in [1f64bce](https://github.com/astronomer/astro-sdk/pull/1183/commits/1f64bce59e2cbe610bf336d00b3ef3d08cb4dd8f)